packages/template-retail-react-app/app/pages/product-list/index.jsx

```diff
@@ -518,7 +517,7 @@ const ProductList = (props) => {
                                           .map((value, index) => (
                                               <ProductTileSkeleton key={index} />
                                           ))
-                                    : productSearchResult.hits.map((productSearchItem) => {
+                                    : productSearchResult?.hits.map((productSearchItem) => {
```

Comment:
@alexvuong with optional chaining you can't have a `.` (as in `hits.map`) after a `?.` (as in ` productSearchResult?.hits.map`)

Pretty sure this needs to be `: productSearchResult?.hits?.map((productSearchItem) => {`

Reply:
@bfeister good catch, fixed as suggested